lib/connection/connection_config.js

```diff
@@ -543,6 +544,16 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     return authenticator;
   };
 
+  /**
+   * Returns the timeout in millis used for authentication by external browser.
+   *
+   * @returns {String}
+   */
+  this.getBrowserActionTimeout = function ()
```

Comment:
is it connection or global config option?

Reply:
It is connection configuration param